core/master/src/main/java/org/drftpd/commands/usermanagement/UserManagementHandler.java

```diff
@@ -1817,12 +1817,27 @@ public CommandResponse doSITE_USER(CommandRequest request) throws ImproperUsageE
 
         Session session = request.getSession();
 
-        if (request.getUser() == null) {
-            return StandardCommandManager.genericResponse("RESPONSE_530_ACCESS_DENIED");
+        // This command can be accessed by anyone, we impose restrictions here to ensure no data is leaked we do not want to be leaked
+        boolean hasPermission = false;
+
+        // Basic check
+        if (request.getUser() != null) {
```

Comment:
Maybe is a naming variable problem but looks like :
> myUser is the user requested by the command for now. But it should be named requestedUser
> reqUser is the current authenticated user that ask for other user information. Should be named currentUser

Reply:
Changed the naming